recipe/build.sh

```diff
@@ -1,9 +1,55 @@
 #!/usr/bin/env bash
 
-./configure --prefix=$PREFIX --with-glib=embedded --enable-nls=no
+if [[ `uname` == 'Linux' ]]; then
+    # don't build mono-helix-client which requires the helix-binaries to build
+    sed -i 's|mono-helix-client||g' mcs/tools/Makefile
+
+    # disable tests
+    sed -i 's|tests unit-tests benchmark||g' mono/Makefile.am
+
+    # use v4.7.1 instead of v4.6
+    # sed -i 's|TARGET_NET_REFERENCE = v4.6|TARGET_NET_REFERENCE = $(BOOTSTRAP_BIN_PROFILE)|g' mcs/tools/upload-to-sentry/Makefile
+
+    # use v4.7.1 instead of v4.7
+    # sed -i 's|BOOTSTRAP_BIN_PROFILE = v4.7|BOOTSTRAP_BIN_PROFILE = v4.7.1|g' mcs/build/profiles/build.make
+
+    # Remove hardcoded lib directory for libMonoPosixHelper.so from the config
+    sed -i 's|$mono_libdir/||g' data/config.in
+
+    # find . -name "*.dll" -not -path "./mcs/class/lib/monolite-linux/*" -not -path "./external/binary-reference-assemblies/v4.7.1/*" -print -delete
+    # find . -name "*.exe" -not -path "./mcs/class/lib/monolite-linux/*" -print -delete
+
+    export CFLAGS="$CFLAGS -fno-strict-aliasing"
+
+    ./autogen.sh --prefix=$PREFIX     \
+                 --with-glib=embedded \
+                 --disable-rpath      \
+                 --with-csc=mcs       \
+                 --disable-system-aot \
+                 --with-moonlight=no
+
+    sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
+    sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
+
+    export LD_LIBRARY_PATH=`pwd`/mono/native/.libs
+    export DYLD_LIBRARY_PATH=`pwd`/mono/native/.libs
```

Comment:
Why do you need these?

Reply:
I copied this from the RPM I found. DYLD path is definitely not needed, I can check if it works wihtut LD_...